drgn/helpers/__init__.py

```diff
@@ -56,3 +57,25 @@ def escape_string(buffer: Iterable[int], escape_single_quote: bool = False,
                                     escape_double_quote=escape_double_quote,
                                     escape_backslash=escape_backslash)
                    for c in buffer)
+
+
+def get_enum(prog: object, name : str, type : str, ignore: Iterable[str] = [],
```

Comment:
`prog: drgn.Program`. Also, let's make this take the type as a `drgn.Type` instead of a string. Then the caller can deal with doing `prog.type` or `prog[...].type_` as appropriate.